Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "avvoenv"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
I think as a new feature (rather than a bug fix) this probably warrants a bump to 0.3.0.